server/src/test/java/org/elasticsearch/node/InternalSettingsPreparerTest.java

```diff
@@ -63,6 +63,7 @@ public void testThatCommandLineArgumentsOverrideSettingsFromConfigFile() throws
         // Value kept from crate.yml
         assertThat(finalSettings.get("path.logs"), Matchers.anyOf(
             is("/some/other/path"),
+            is("C:\\some\\other\\path"),
             is("D:\\some\\other\\path")
```

Comment:
More safe?

```suggestion
            containsString("\\some\\other\\path")
```

Reply:
or ```endsWith(":\\some\\other\\path") ?
```

Reply:
Thank you for your suggestions. I've just updated the patch with @matriv's one.